packages/dd-trace/test/guardrails/telemetry.spec.js

```diff
@@ -16,7 +18,7 @@ describe('sendTelemetry', () => {
 
   beforeEach(() => {
     cleanup = telemetryForwarder()
-    sendTelemetry = proxyquire('../src/guardrails/telemetry', {})
```

Comment:
This is pretty weird. It looks like the old proxyquire call here was wrong as that file doesn't exist relative to this file, and that you fixed it. And it's not like the location of this test file moved.

Did you find that running this test file would throw errors and that when you fixed the path it started working? If so we might have a bug in our test suite where we're not properly executing this test file...

Reply:
Just back from PTO. I assume you got this all figured out?